@navikt/core/react/src/modal/Modal.tsx

```diff
@@ -143,8 +144,10 @@ export const Modal = forwardRef<HTMLDialogElement, ModalProps>(
 
     useBodyScrollLock(modalRef, portalNode, isNested);
 
-    const isWidthPreset =
-      typeof width === "string" && ["small", "medium"].includes(width);
+    let isWidthPreset = false;
+    if (typeof width === "string" && ["small", "medium"].includes(width))
+      isWidthPreset = true;
+    else if (size === "small") isWidthPreset = true;
```

Comment:
Think it would make sense to separate `width` and `size` functionality. So width handles modal width like today and size just handles internal spacing/padding.